tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipsePitSummaryViewTest.java

```diff
@@ -0,0 +1,84 @@
+package org.pitest.pitclipse.ui.tests;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.swtbot.swt.finder.junit.SWTBotJunit4ClassRunner;
+import org.eclipse.swtbot.swt.finder.utils.SWTBotPreferences;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pitest.pitclipse.ui.behaviours.pageobjects.PitSummaryView;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.not;
+import static org.pitest.pitclipse.ui.behaviours.pageobjects.PageObjects.PAGES;
+
+import java.io.File;
+
+/**
+ * @author Jonas Kutscha
+ * 
+ */
+@RunWith(SWTBotJunit4ClassRunner.class)
+public class PitclipsePitSummaryViewTest extends AbstractPitclipseSWTBotTest {
+    private static final String TEST_PROJECT = "org.pitest.pitclipse.testprojects.twoclasses";
+    private static final String FOO_BAR_PACKAGE = "foo.bar";
+    private static final String FOO_CLASS = "Foo";
+    private static final String TEST_PROJECT_WITH_DIFFERENT_SRC = "org.pitest.pitclipse.testprojects.differentSrc";
```

Comment:
Please don't use capital letters in project names (nor in the project directory).
Please make sure you change consistently the name and the directory to `org.pitest.pitclipse.testprojects.differentsrc`